lms/templates/verify_student/pay_and_verify.html

```diff
@@ -65,7 +65,6 @@
       data-platform-name='${platform_name}'
       data-course-key='${course_key}'
       data-course-name='${course.display_name}'
-      data-course-start-date='${course.start_datetime_text()}'
```

Comment:
What's consuming this attribute, if anything? Due to the fact that verification is (strangely) tied to payment, this code may still be in use.